lib/Controller/ApiController.php

```diff
@@ -20,13 +20,19 @@
  */
 
 
-namespace OCA\ServerInfo;
+namespace OCA\ServerInfo\Controller;
 
 use OCP\AppFramework\Http\DataResponse;
 use OCP\AppFramework\OCSController;
 
 class ApiController extends OCSController {
 
+
+	public function __construct($appName,
```

Comment:
It seems the AppFramework needs this.
We should fix it properly. But for now this workaround does the job.